.github/workflows/circt.yml

```diff
@@ -0,0 +1,23 @@
+name: Build and push CIRCT docker image
+
+# Run on request and every day at 0300 PT
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 11 * * *
+
+jobs:
+  build-circt-image:
+    name: Build and push Docker image containing built CIRCT utilities
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get CIRCT images repo
+        uses: actions/checkout@v2
+      - name: Build and push image
+        working-directory: ./circt
+        run: |
+          sudo apt-get install -y jq
```

Comment:
I may have gotten this wrong. It may just be `apt install -y jq`. However, if this fails the GH action, I can fix it up.